app/templates/base.html

```diff
@@ -120,15 +120,15 @@ <h1 id="error-title" class="card-title">{% block error_title %}Error{% endblock
 </footer>
 {% endspaceless %}
 
-<script>
+<script defer>
 {% include "app/js/page-status.js" %}
 </script>
 {% if debug %}
     {% load django_htmx %}
     {% django_htmx_script %}
 {% endif %}
 {% if debug_toolbar %}
-<script>
+<script defer>
```

Comment:
According to MDN:
> This attribute must not be used if the src attribute is absent (i.e. for inline scripts), in this case it would have no effect.

https://developer.mozilla.org/en-US/docs/Web/HTML/Element/script#defer

Reply:
Thanks. Good catch. My concern here was that these scripts at the end of the page (particularly page-status.js) relies on htmx, and deferred scripts are executed in order. With HTMX deferred, we need to know that page-status.js is still executed after HTMX, since it uses `htmx.on()`.  I like inlining the JS to avoid a round-trip, but loading it with `src` (and `defer`) seems to be the only way to go. The performance of the debugging scripts aren't a big issue.

Is it safe to use `document.body.addEventListener('htmx:load', ...)` instead of `htmx.on()` in page-status.js and thereby maintain the current inline script?